oauth2_http/javatests/com/google/auth/oauth2/ImpersonatedCredentialsTest.java

```diff
@@ -908,6 +908,38 @@ public void sign_serverError_throws() {
     }
   }
 
+  @Test
+  public void sign_sameAs_nonGDU() {
```

Comment:
Is it possible for there be a test for ImpersonatedCredentials `sign_universeDomainException()`?

Reply:
added in [083557d](https://github.com/googleapis/google-auth-library-java/pull/1566/commits/083557d8f8a0a655b3a785e222816b58b193591c)